.circleci/config.yml

```diff
@@ -769,8 +769,25 @@ jobs:
             aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
             cd ../..
 
+            echo -e "Here is the link to download the wheel file: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
             echo -e "https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}" >> S3_URL
 
+      - run:
+          name: Upload wheel for Metrics App!
```

Comment:
nit: I feel like this name is too specific. Could this instead be something like "Upload release/demo wheel" since in theory it could be used for more than just our internal metrics app